docs/userGuide/puml.md

```diff
@@ -15,25 +15,31 @@
 
 <box type="warning">
 
-**[Java](https://www.java.com/en/download/) must be installed to use this feature**. _(Any recent version should work)_
-</box>
-<box type="warning">
+**[<tooltip content="Any recent version">Java</tooltip>](https://www.java.com/en/download/) must be installed to use this feature**
 
-**[Graphviz](https://www.graphviz.org/download/) must be installed to generate diagrams**, with the exception of sequence and activity diagrams. (_tested with v2.38_)
+**[<tooltip content="Tested with v2.38">Graphviz</tooltip>](https://www.graphviz.org/download/) must be installed to generate <tooltip content="with the exception of sequence and activity diagrams">diagrams</tooltip>**
```

Comment:
Will there be a case where activity and sequence diagrams don't work if they used a certain syntax for another component (without Graphviz) ?

Reply:
Honestly, I don't know. What do you mean by a certain syntax for another component?

Reply:
On a separate note, use tooltips only in cases where you want to supply a definition/explanation of a term. Otherwise, the reader will have to retrieve every tooltip to see if there is something relevant in the tooltip. Use dimmed text to provide unimportant details.
`recent version` is not precise enough. Say something like `version 8 or later`.